tests/unit/amo/components/TestInstallButtonWrapper.js

```diff
@@ -671,9 +622,8 @@ describe(__filename, () => {
       });
 
       it('has the expected button text for a theme, which is incompatible', () => {
-        renderAsIncompatible({
-          addon: createInternalAddonWithLang(fakeTheme),
-        });
+        addon = { ...fakeTheme };
```

Comment:
This could either be `addon = fakeTheme;` or `addon.type = ADDON_TYPE_STATIC_THEME`.